src/components/blog/FeaturePost.js

```diff
@@ -1,34 +1,84 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import Image from 'gatsby-image';
 
 import Heading from '../Heading';
 import Text from '../Text';
+import { ReactComponent as Arrow } from '../../assets/arrow.svg';
+import Link from '../Link';
 
-import AuthorBox from './AuthorBox';
-import PostDetailsBox from './PostDetailsBox';
-import PostLink from './PostLink';
+const Container = styled.div`
+  height: 500px;
```

Comment:
Q: Why fix height like this ? I think it is better to keep the height of this section fluid and try to match design specs using combination of margin and padding.
If it is really necessary, consider using `rem` units

Reply:
Yea, px was supposed to be a temporary solution. I forgot to delete them.